swagger_server/models/collectioninstrument.py

```diff
@@ -0,0 +1,153 @@
+# coding: utf-8
+
+from __future__ import absolute_import
+from swagger_server.models.classifiers import Classifiers
+from .base_model_ import Model
+from datetime import date, datetime
+from typing import List, Dict
+from ..util import deserialize_model
+
+
+class Collectioninstrument(Model):
+    """
+    NOTE: This class is auto generated by the swagger code generator program.
+    Do not edit the class manually.
+    """
+    def __init__(self, classifiers: Classifiers=None, survey_id: str='urn:ons.gov.uk:id:survey:001.234.56789', ci_type: str='OFFLINE', id: str='urn:ons.gov.uk:id:ci:001.234.56789'):
```

Comment:
This comes from the original API specification, this does need a tidy up, but it's not in-scope for the moment. (it's auto-generated and overridden)